lib/commands/command.bash

```diff
@@ -84,6 +88,9 @@ _asdf_cached_envrc() {
 }
 
 _asdf_envrc() {
+  if [ -n "$ASDF_DIRENV_DEBUG" ]; then
+    echo 'set -x'
+  fi
```

Comment:
This introduces a minor bug, see https://github.com/asdf-community/asdf-direnv/pull/111 for a fix.